src/store/index.jsx

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import reducers from '../reducer/index';
+
+const enhacers = compose(
+  applyMiddleware(thunk),
```

Comment:
Good work @Emmanuelaaron, 
Please help me understand what thunk does?
Thanks

Reply:
@jennizalwango you may visit this link https://medium.com/fullstack-academy/thunks-in-redux-the-basics-85e538a3fe60 to understand what **thunk** does.

Reply:
@jennizalwango, Redux -thunk is a middleware for redux that allows us to directly access the dispatch method.

Reply:
okay thanks, @Emmanuelaaron,
I think now you are good to go

Reply:
Thank you all, I also didn't know what it does.